diff --git a/windows/CMakeLists.txt b/windows/CMakeLists.txt
index c8228bc..f59c43d 100644
--- a/windows/CMakeLists.txt
+++ b/windows/CMakeLists.txt
@@ -33,7 +33,7 @@ endif()
 set_target_properties(hidapi_winapi
     PROPERTIES
         EXPORT_NAME "winapi"
-        OUTPUT_NAME "hidapi"
+        OUTPUT_NAME "hidapi64"
         VERSION ${PROJECT_VERSION}
         PUBLIC_HEADER "${HIDAPI_PUBLIC_HEADERS}"
 )
 